src/NServiceBus.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.24720.0
+VisualStudioVersion = 14.0.25123.0
```

Comment:
No, but given that VS will want to update that every time someone does something that touches the solution file anyway, I didn't see the harm in letting it update it.

I can remove it if you'd prefer.